src/Sylius/Bundle/AdminBundle/Resources/private/js/app.js

```diff
@@ -14,7 +14,7 @@
             'containerSelector': '#sylius_calculator_container'
         });
 
-        $('#sylius_customer_create_user').change(function(){
+        $('#sylius_customer_createUser').change(function(){
```

Comment:
Because we use camelCase to named field in FormType. So we have sylisus_nameOfEntity_nameOfField. Before this field was in twig template and it has the id "sylius_customer_create_user".